Vagrantfile

```diff
@@ -0,0 +1,34 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant::Config.run do |config|
+	# Base box to build off, and download URL for when it doesn't exist on the user's system already
+	config.vm.box = "ubuntu/trusty32"
```

Comment:
docker use 16.04. I believe we should use consistend image here.

It looks that newest ubuntu images are stored on Atlas: https://atlas.hashicorp.com/ubuntu